packages/wallet-adapter-react/docs/BYO-wallet-selector.md

```diff
@@ -0,0 +1,205 @@
+# Build Your Own Wallet Selector
```

Comment:
we probably want to include it or at least mention it on aptos.dev cc @jmintuitive